xcube/server/testing.py

```diff
@@ -65,6 +66,14 @@ def setUp(self) -> None:
 
         self.http = urllib3.PoolManager()
 
+        while True:
+            # noinspection PyBroadException
+            try:
+                self.fetch('/')
+                break
+            except Exception:
+                time.sleep(0.1)
```

Comment:
```suggestion
                # Intermittent MaxRetryError (and others) in CI unit test runs
                # https://github.com/dcs4cop/xcube/issues/899
                time.sleep(0.1)
```

Reply:
Added reference to the issue before the respective code block.